packages/react/src/components/checkbox/checkbox.react.stories.tsx

```diff
@@ -0,0 +1,81 @@
+import { Checkbox, CheckboxProps, HelperText } from '@onfido/castor-react';
+import React, { ChangeEvent, useRef } from 'react';
+import { Meta, omit, Story, storyOf } from '../../../../../docs';
+
+export default {
+  title: 'React/Checkbox',
+  component: Checkbox,
+  argTypes: {
+    ...omit<CheckboxProps>('className', 'style'),
+    children: { control: 'text' },
+    bordered: {
+      table: { type: { summary: 'boolean' } },
+    },
+    invalid: {
+      table: { type: { summary: 'boolean' } },
+    },
+    disabled: {
+      table: { type: { summary: 'boolean' } },
+    },
+  },
+  args: {
+    bordered: false,
+    invalid: false,
+    disabled: false,
+  },
+} as Meta<CheckboxProps>;
+
+export const Playground: Story<CheckboxProps> = (props: CheckboxProps) => (
+  <Checkbox {...props} />
+);
+
+export const Bordered = storyOf(Checkbox, 'bordered', [true, false], {
+  labelMode: 'children',
+});
+Bordered.argTypes = omit<CheckboxProps>('bordered');
+Bordered.args = { name: 'bordered-playground' };
```

Comment:
💅🏻  [nitpick]
I don't think any of these stories need a `name`.
Especially since they're shared between each checkbox when they should be unique.